component/admin/language/en-GB.com_jevents.ini

```diff
@@ -1251,4 +1251,6 @@ JEV_ERROR_NOT_AUTH_CSS="You are not authorised to use the Custom CSS Tool. Only
 
 JEV_MODULE_CONFIG="Module Config"
 JEV_ORPHAN_EVENTS_DETECTED_AND_PLACED_IN_SPECIAL_ORPHAN_EVENT_CATEGORY_OR_CALENDAR="Orphan events have been detected and repaired.<br/><br/>These have been placed in a special category or calendar called 'Orphans' so that you can edit or remove them as required.<br/>Please note that this special orphan category and calendar is unpublished by default. <br/><br/>If orphan events have been found we apologise for any inconvenience caused."
-JEV_DUPLICATE_EXTENSION_WARNING="We have duplicate entries in the extensions table.  These are being cleaned up.  <br/><br/><strong>Please check your configuration settings and save them</strong>"
\ No newline at end of file
+JEV_DUPLICATE_EXTENSION_WARNING="We have duplicate entries in the extensions table.  These are being cleaned up.  <br/><br/><strong>Please check your configuration settings and save them</strong>"
+
+JEV_FALLBACK_CATEGORY="Fallback Category"
```

Comment:
I would then change this to then 

'Default / Override Category'

As it's used if there are no categories set too.